tests/load-tests/pkg/journey/handle_component.go

```diff
@@ -104,7 +105,18 @@ func createComponent(f *framework.Framework, namespace, name, repoUrl, repoRevis
 		},
 	}
 
-	_, err := f.AsKubeDeveloper.HasController.CreateComponent(componentObj, namespace, "", "", appName, false, annotationsMap)
+        err := retry.Do(
```

Comment:
Hello! Maybe just reporting a bug would be better? If this happens to us, it can happen to customers as well and our goal is to ensure there are no scaling issues. I'm very sorry it took me so long to make my mind :-/

Reply:
Ok, let's file a bug and close this PR.